brave/brave-core
# Removes date from ads grants in the wallet

Resolves https://github.com/brave/brave-browser/issues/4169

UI PR: https://github.com/brave/brave-ui/pull/458

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- enable rewards
- view some ads, trigger ads grant (or recover wallet with ads grant)
- claim ads grant
- claim ugp grant
- make sure that you see something like this (first one is ads, second one is ugp)
![image](https://user-images.githubusercontent.com/9574457/56759370-1b77bd00-6799-11e9-8016-2f0b3973c7a2.png)


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
